lib/taste_tester/commands.rb

```diff
@@ -294,19 +294,24 @@ def self._find_impact(changeset)
 
       cookbooks = Set.new(changeset.cookbooks)
       roles = Set.new(changeset.roles)
+      databags = Set.new(changeset.databags)
 
       if cookbooks.empty? && roles.empty?
         logger.warn('No cookbooks or roles have been modified.')
         return []
       end
 
       unless cookbooks.empty?
-        logger.warn('Modified Cookbooks:')
-        cookbooks.each { |cb| logger.warn("\t#{cb}") }
+        logger.info('Modified Cookbooks:')
+        cookbooks.each { |cb| logger.info("\t#{cb}") }
       end
       unless roles.empty?
-        logger.warn('Modified Roles:')
-        roles.each { |r| logger.warn("\t#{r}") }
+        logger.info('Modified Roles:')
+        roles.each { |r| logger.info("\t#{r}") }
+      end
+      unless databags.empty?
+        logger.info('Modified Databags:')
+        databags.each { |db| logger.info("\t#{db}") }
```

Comment:
You're 100% correct databags are not gonna help with the dependency tree (they're dynamically called from recipes), however, since you have this code here which prints all "modified" things, it seems useful to support databags (as you've added here)